mkosi/resources/mkosi-initrd/mkosi.conf

```diff
@@ -1,8 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 [Output]
-ImageId=initrd
```

Comment:
Why not `@ImageId=initrd` instead?

Reply:
Because that doesn't seem like that useful an image ID. It doesn't really identify anything